Cython/Compiler/ModuleNode.py

```diff
@@ -233,8 +233,30 @@ def h_entries(entries, api=0, pxd=0):
                 h_code.putln('#error "Unicode module names are not supported in Python 2";')
             h_code.putln("PyMODINIT_FUNC init%s(void);" % py2_mod_name)
             h_code.putln("#else")
-            h_code.putln("PyMODINIT_FUNC %s(void);" % self.mod_init_func_cname('PyInit', env))
-            h_code.putln("#endif")
+            py3_mod_func_name = self.mod_init_func_cname('PyInit', env)
+            warning_string1 = EncodedString('Use PyImport_AppendInittab("%s", %s)' % (
+                                    py2_mod_name, py3_mod_func_name))
+            warning_string2 = 'instead of calling %s directly.' % py3_mod_func_name
+            h_code.putln('/* WARNING: From Python 3.5 %s */' % warning_string1)
+            h_code.putln("/* %s */" % warning_string2)
```

Comment:
Can't we keep the two warning strings together? `textwrap.wrap()` can do the line wrapping for you if you want separate comment lines.